src/SimulationService.jl

```diff
@@ -232,6 +233,15 @@ function modelEquation(::HTTP.Request, id::String)
     ])
 end
 
+# POST /model-equation
+function modelToEquation(req::HTTP.Request)
+    sys = amr_get(req.body, ODESystem)
+    model_latex = latexify(sys)
+    return Dict([
+        (:latex, model_latex.s)
+    ])
+end
```

Comment:
Erm ... this doesn't work??? Did you miss a commit?

See error below.

```
┌ Error: ERROR:
│   exception =
│    MethodError: no method matching amr_get(::Vector{UInt8}, ::Type{ModelingToolkit.ODESystem})
│
│    Closest candidates are:
│      amr_get(::JSON3.Object, ::Type{ModelingToolkit.ODESystem})
│       @ SimulationService ~/workspace/askem/sciml-service/src/operations.jl:9
│
│    Stacktrace:
│      [1] modelToEquation(req::HTTP.Messages.Request)
│        @ SimulationService ~/workspace/askem/sciml-service/src/SimulationService.jl:238
│      [2] #6
│        @ Oxygen.Core.Handlers ~/.julia/packages/Oxygen/LWjgB/src/handlers.jl:40 [inlined]
│      [3] #14#15
│        @ Oxygen.Core.Handlers ~/.julia/packages/Oxygen/LWjgB/src/handlers.jl:54 [inlined]
│      [4] #14
│        @ Oxygen.Core.Handlers ~/.julia/packages/Oxygen/LWjgB/src/handlers.jl:53 [inlined]
```

Reply:
To be honest, I haven't tested it yet, I was going to ask you how to run this services locally yesterday, but missed the boat. Let me figure that out and get back to you.

Reply:
You probably need something like:
```
amrJSON = JSON3.read(req.body)
```